CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change Log
+
+## [1.0.12]
+
+* Changes from April 2022 the last time this extension was released. 
```

Comment:
💡 - if changelog will lead to the release - might be lets update changelog as separate PR - and name it as release, as after changelog - next we could do is release.e

Reply:
@Tatsinnit Plan was not to add release just a placeholder, I can remove it and add another PR for changelog.